### DISCUSSION TITLE
Not sure where to report possible bug:

### DISCUSSION THREAD

**Discussion by kstrib** [nalexn/ViewInspector#252] [General]
![Screenshot 2023-07-20 at 8 31 45 PM](https://github.com/nalexn/ViewInspector/assets/26911074/81d9c907-31d8-4886-a586-18bb3bb76517)

In the screenshot, you'll see the gray error box was a previous error, that told me not to look for RoundedRectangle inside of a shape by using: `.find(RoundedRectangle.self)` and to use the suggested line of code instead.

When I used the suggested line of code, I received an error response stating that it was impossible because it has no clue what I'm talking about. 🤷🏾‍♂️

Any ideas on how to proceed? Is it me, am I doing something wrong? Or is this a bug?

- kstrib

**Comment by nalexn** [comment]
Hey, you should use `.find(ViewType.Shape.self)` for that, also more info here #158
